frame_id: "world_origin"

body:
  frame_id: "fcu"

timer_main:
  rate: 10.0 # [Hz]

# School
bounded_box:
  min_x: -25.0 # [m]
  max_x: 25.0 # [m]
  min_y: -20.0 # [m]
  max_y: 15.0 # [m]
  min_z: 1.0 # [m]
  max_z: 20.0 # [m]

# Police Station
#bounded_box:
  #min_x: -15.0 # [m]
  #max_x: 15.0 # [m]
  #min_y: -20.0 # [m]
  #max_y: 15.0 # [m]
  #min_z: 1.0 # [m]
  #max_z: 20.0 # [m]

# Maze
#bounded_box:
  #min_x: -10.0 # [m]
  #max_x: 10.0 # [m]
  #min_y: -9.0 # [m]
  #max_y: 9.0 # [m]
  #min_z: 0.2 # [m]
  #max_z: 2.5 # [m]

uav_parameters:
  max_vel: 1.0 # [m/s]
  max_accel: 1.0 # [m/s^2]
  max_heading_vel: 2.0 # [rad/s]
  max_heading_accel: 2.0 # [rad/s^2]

local_planning:
  N_max: 50
  N_termination: 300
  N_yaw_samples: 15
  radius: 3.0 # [m]
  step_size: 2.0 # [m]
  tolerance: 2.0 # [m]
  g_zero: 5.0

global_planning:
  N_min_nodes: 150
  global_max_acceleration_iterations: 2

camera:
  frame_id: "rgbd_front_pitched/aligned_depth_to_color"
  h_fov: 1.51844 # [rad]
  width: 1080
  height: 720
  min_distance: 0.2 # [m]
  max_distance: 5 # [m]

path:
  uav_radius: 1.5 # [m]
  lambda: 0.2 # (tunning factor)
  lambda2: 0.5 # (tunning factor)
  global_lambda: 0.02 # (tunning factor)
  global_lambda2: 0.05 # (tunning factor)
  max_acceleration_iterations: 3

